restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -145,20 +141,8 @@ public boolean treatBooleansAsNumeric() {
     return !dataStore.supportsSecondaryIndex();
   }
 
-  public static List<String> getForbiddenCharactersMessage() {
-    return forbiddenCharacters.stream().map(ch -> "`" + ch + "`").collect(Collectors.toList());
-  }
-
-  public static boolean containsIllegalChars(String x) {
-    return forbiddenCharacters.stream().anyMatch(ch -> x.indexOf(ch) >= 0);
-  }
-
-  public static String replaceIllegalChars(String x) {
-    String newStr = x;
-    for (Character y : forbiddenCharacters) {
-      newStr = newStr.replace(y, '_');
-    }
-    return newStr;
+  public static boolean containsIllegalSequences(String x) {
```

Comment:
Not directly related to changes here, but is there a specification of how handling should work? Like a bigger document that describes intended behavior, including disallowed characters and so on?
And if there isn't, would it make to start writing/maintaining such document(s) now? :)

Reply:
Yea I think we are getting to that point; there is so document but it would be good to have.